FrameworkSystem/private/standardLogging/test/TestLogUtilities.py

```diff
@@ -0,0 +1,90 @@
+"""
+Test Logger Wrapper
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+from StringIO import StringIO
+
+from DIRAC.FrameworkSystem.private.standardLogging.LoggingRoot import LoggingRoot
+from DIRAC.FrameworkSystem.private.standardLogging.Logging import Logging
+
+
+gLogger = LoggingRoot()
+
+
+def cleaningLog(log):
+  """
+  Remove date and space from the log string
+  """
+  log = log.split('UTC ')
+  if len(log) > 1:
```

Comment:
can't you replace all that with `log = log[-1]` ?

Reply:
Yes, I'm going to change that.
Thanks for the review